coordinator/src/services/translation/common/index.ts

```diff
@@ -174,3 +164,33 @@ export function getNumericValueAsString(numericValue: string | number | Lossless
     return numericValue.toString()
   }
 }
+
+function isStringParameter(parameter: fhir.Parameter): parameter is fhir.StringParameter {
+  return (parameter as fhir.StringParameter).valueString !== undefined
+}
+
+function isIdentifierParameter(parameter: fhir.Parameter): parameter is fhir.IdentifierParameter {
+  return (parameter as fhir.IdentifierParameter).valueIdentifier !== undefined
+}
+
+export function getStringParameterByName(
+  parameters: Array<fhir.ParameterTypes>,
+  name: string,
+  fhirPath = ""
+): fhir.StringParameter {
+  return onlyElement(parameters.filter(isStringParameter).filter(parameter => parameter.name === name),
+    fhirPath,
```

Comment:
I think the FHIR path will always be `Parameters.parameter` and doesn't need to be passed in